traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -513,8 +514,9 @@ public Geolocation getLocation(final String clientIP, final DeliveryService deli
 			}
 		}
 
-		if (clientLocation.isDefaultLocation() && defaultGeolocationsOverride.containsKey(clientLocation.getCountryCode())) {
-			clientLocation = defaultGeolocationsOverride.get(clientLocation.getCountryCode());
+		if (clientLocation.isDefaultLocation() && getDefaultGeoLocationsOverride().containsKey(clientLocation.getCountryCode())) {
+			clientLocation = deliveryService.getMissLocation();
```

Comment:
What happens if `missLocation` is null? 

I think the most consistent behavior for a null `missLocation` would be to continue using 
```
defaultGeolocationsOverride.get(clientLocation.getCountryCode());
````